charts/velero/ci/test-values.yaml

```diff
@@ -1,3 +1,9 @@
+# Set provider name and backup storage location bucket name
+configuration:
```

Comment:
Is this configuration strictly necessary to test the chart now?

Reply:
Yes, because these are mandatory parameters in the structural schema, Orherwise, helm install would failed.